### PR TITLE
Docs: Update hyperparameter tuning with optuna notebook.

### PR DESCRIPTION
**Pull request by npnv**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

**Comment by jaheba** [comment]
@npnv I'm curious, what is different from the existing tutorial?

**Comment by npnv** [comment]
@jaheba The difference mainly lies in using the newly modified `split` function instead of self-defined data split function.

File: docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template [hunk 1]
```diff
@@ -66,59 +66,56 @@ plt.show()
 ```python
 import optuna
 import torch
-from gluonts.torch.model.deepar import DeepAREstimator
-from gluonts.mx import Trainer
+from gluonts.dataset.split.splitter import split
 from gluonts.evaluation import Evaluator
+from gluonts.mx import Trainer
+from gluonts.torch.model.deepar import DeepAREstimator
```
Comment (jaheba) [inline docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template]: This looks wrong -- `torch` and `mx` shouldn't mix.

Looking at the docs, `Trainer` seems unused?

Comment (lostella) [reply docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template]: yes, the import of `Trainer` should be removed

Comment (npnv) [reply docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template]: Will do.

File: docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template [hunk 2]
```diff
@@ -66,59 +66,55 @@ plt.show()
...
-from gluonts.mx import Trainer
+from gluonts.dataset.split.splitter import split
 from gluonts.evaluation import Evaluator
+from gluonts.torch.model.deepar import DeepAREstimator
 ```
 
 We will now tune the DeepAR estimator on our training data using Optuna. We choose two hyperparameters `num_layers` and `hidden_size` to optimize.
 
-First, we define an `DeepARTuningObjective` class used in tuning process of Optuna.
+First, we define a `dataentry_to_dataframe` method to transfer the DataEntry into DataFrame.
```
Comment (lostella) [inline docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template]:
```suggestion
First, we define a `dataentry_to_dataframe` method to transform a `DataEntry` into a `pandas.DataFrame`.
```

File: docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template [hunk 3]
```diff
@@ -66,59 +66,55 @@ plt.show()
...
-- `entry_past`: the training part 
-- `entry_future`: the label part used in validation
 In the `__call__` method, we define the way the `DeepAREstimator` is used in training and validation.
 
 
 ```python
+def dataentry_to_dataframe(entry):
+    df = pd.DataFrame(
+        entry['target'],
+        columns=[entry['item_id']],
```
Comment (lostella) [inline docs/tutorials/advanced_topics/hp_tuning_with_optuna.md.template]: Maybe use `.get` here in case the entry has no `item_id`

**Review by lostella — CHANGES_REQUESTED** [review]

**Review by lostella — APPROVED** [review]
